components/admin/lessons/AdminLessonCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Link from 'next/link'
+import styles from '../../../scss/adminLessonCard.module.scss'
+import { Lesson } from '../../../graphql/'
+
+type Props = {
+  lesson: Lesson
+  pendingFlaggedQuestions: number
+}
+
+export const AdminLessonCard: React.FC<Props> = ({
+  lesson,
+  pendingFlaggedQuestions
+}) => {
+  const { title, description, docUrl } = lesson
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.container__heading}>{title}</div>
+
+      <div className={styles.container__description}>
+        {description} The Dojo is a space for you to store your coding journal
```

Comment:
`The Dojo is a space for you to store your coding journal and keep track of your learnings` <- take this out please, I think we only want to display the description